lms/djangoapps/django_comment_client/management/commands/tests/test_rename_user.py

```diff
@@ -0,0 +1,128 @@
+            "closed": False,
+            "comment_count": 9,
+            "commentable_id": "video_1",
+            "course_id": "MITx/6.002x/2012_Fall",
+            "created_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "historical_abuse_flaggers": [],
+            "last_activity_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "tags_array": [
+                "c#",
+                "2012",
+                "java-sucks"
+            ],
+            "thread_type": "discussion",
+            "title": "Dolore et molestias quia maxime eaque quos voluptatem et.",
+            "updated_at": datetime.datetime.strptime('22092013', "%d%m%Y"),
+            "visible": True,
+            "votes": {
+                "count": 7,
+                "down": [
+                    "1",
+                    "3",
+                    "4",
+                    "5",
+                    "7"
+                ],
+                "down_count": 5,
+                "point": -3,
+                "up": [
+                    "2",
+                    "6"
+                ],
+                "up_count": 2
+            }
+
+        }
+        cls.mongo_comment_ids.append(cls.db.contents.insert(comment_1))
+
+    @classmethod
+    def seed_mongo_user(cls):
+        '''
+        Seeds 3 mongo test users for the tests that follow
+        '''
+        user_1 = {
+            "default_sort_key": "date",
+            "notification_ids": [],
+            "external_id": "test_user_1",
+            "username": cls.sql_user.username
+        }
+        cls.mongo_user_ids.append(cls.db.users.insert(user_1))
+
+    def test_data_seed(self):
+        self.assertTrue(len(self.mongo_user_ids) == 1, 'Test mongo users were not properly seeded, rendering tests invalid.')
+        self.assertTrue(len(self.mongo_comment_ids) == 1, 'Test mongo comments were not properly seeded, rendering the tests invalid.')
+
+    def test_incorrect_num_args(self):
+        zero_args = []
+        one_arg = ['arg']
+        three_args = ['arg', 'arg', 'arg']
+        self.assertRaises(CommandError, self.rename_cmd.handle, *zero_args)
+        self.assertRaises(CommandError, self.rename_cmd.handle, *one_arg)
+        self.assertRaises(CommandError, self.rename_cmd.handle, *three_args)
+
+    def test_successful_change(self):
+        saved_stdout = sys.stdout
+        try:
+            out = StringIO()
+            sys.stdout = out
+            args = [self.sql_user.username, 'someArbitraryUserName']
+            self.rename_cmd.handle(*args)
+            output = out.getvalue().strip()
+            self.assertRegexpMatches(output, 'changed username of user:')
+        finally:
+            sys.stdout = saved_stdout
+
+    @classmethod
+    def tearDownClass(cls):
+        # remove test mongo users
+        for test_user_id in cls.mongo_user_ids:
+            cls.db.users.remove({"_id": test_user_id})
```

Comment:
running 
cls.db.users.remove({})
would be problematic because it would remove all the users inside the client's forum database and they might depend on this data for their dev environment. This is why I remove only the users that I add, so as not to alter any of their existing data.